frontend/public/components/cluster-settings/cluster-settings-utils.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-undef */
+import * as React from 'react';
+import { Dropdown } from '../utils';
+import * as classNames from 'classnames';
+
+export const MappingMethod: React.FC<MappingMethodProps> = ({selectedMappingMethod, onChange, required=true}) => {
```

Comment:
```suggestion
export const MappingMethod: React.SFC<MappingMethodProps> = ({selectedMappingMethod, onChange, required=true}) => {
```

Reply:
@spadgett i believe SFC is deprecated, and FC is now suggested. Keep SFC?

Reply:
Didn't realize that! Please use the non-deprecated one :)